pkg/volume/azure_dd/vhd_util.go

```diff
@@ -1,145 +0,0 @@
-/*
```

Comment:
this has moved into the common file of the disk controllers. Since this only needed by Blob Disk + Managed Disk (and will *always* be, since any block device based disks will always use AzureDiskVolumeSource type with the addition of Kind parameter) also they have nothing to do with "vhd" per-se